spring-integration-azure/spring-integration-eventhub/src/main/java/com/microsoft/azure/spring/integration/eventhub/util/EventDataHelper.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See LICENSE in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.spring.integration.eventhub.util;
+
+import com.microsoft.azure.eventhubs.EventData;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+public class EventDataHelper {
+
+    public static String toString(EventData eventData){
```

Comment:
`EventData` is interface, and you would like not to put this `toString()` to interface, as it just for logging, right ?

Reply:
`EventData` is provided by event hub SDK, we have no control.